PermanentOrg/web-app
# Upgrade to Angular 13

This PR upgrades our Angular version from Angular 12 to Angular 13. This includes the new Ivy renderer and other fancy stuff, yay! This PR also upgrades the various peer dependencies used in our project to be compatible with the core upgrades to Angular 13. Much like the other upgrade PRs, this also includes the automatic migrations run by the Angular CLI as well as manual migrations that fix things the CLI didn't.

To test:
- Make sure to run `npm install`!
- Run tests, the local dev proxy, and a production build
- Make sure everything works
- After testing, if you're going back to a branch on Angular 12, the `.angular/cache` directory can be safely deleted. (It's only an Angular 13 thing, so when you switch you're going to notice it'll suddenly be tracked by git!)

Resolves PER-8854 (Upgrade to Angular 13). Also clears out the rest of our current dependabots: Resolves #109 and Resolves #132!
